man/pam_systemd.xml

```diff
@@ -262,6 +262,54 @@
     based on the <varname>$DISPLAY</varname> variable.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Session limits</title>
+
+    <para>PAM modules earlier in the stack, that is those that come before <command>pam_systemd.so</command>,
+    can set session scope limits using the PAM context objects. The data for these objects is provided as NUL-terminated C strings
+    and maps directly to the respective unit resource control directives. See
+    <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry> for more information about the resources.
+    Also, see <citerefentry><refentrytitle>pam_set_data</refentrytitle><manvolnum>3</manvolnum></citerefentry> for additional information about how to set
+    the context objects.
+    </para>
```

Comment:
i think it would be make sense to explicitly clarify that these limits are *per* session, and do not effect the user's processes a whole, and in particular do not affect the systemd --user instance, as that's a singleton services shared by all sessions of a user. i.e. a sentence like this:

```
Note that these limits apply on individual sessions of the user, they do not apply to all user processes as a whole together. Note that in particular the per-user `user@.service` unit instance (which runs the `systemd --user` manager process and its children) is not covered by this limit.
```